File: onclick/app/src/main/java/com/example/android/onclick/MainActivity.java
package com.example.android.onclick;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        NumberClickListener clickListener=new NumberClickListener();
        TextView numbers=(TextView)findViewById(R.id.numbers);
        numbers.setOnClickListener(clickListener);
    }
}

File: ScoreBoardLogin/app/src/main/java/com/example/android/scoreboard_login/MainActivity.kt
package com.example.android.scoreboard_login

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.AdapterView
import androidx.databinding.DataBindingUtil
import com.example.android.scoreboard_login.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity(){
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        @Suppress("UNUSED_VARIABLE")
        val binding = DataBindingUtil.setContentView<ActivityMainBinding>(this, R.layout.activity_main)
    }
}